docs/enhancements/template.md

```diff
@@ -0,0 +1,154 @@
+---
+title: neat-enhancement-idea
+authors:
+  - "@janedoe"
+reviewers:
+  - TBD
+  - "@alicedoe"
+approvers:
+  - TBD
+  - "@oscardoe"
+creation-date: yyyy-mm-dd
+last-updated: yyyy-mm-dd
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced|informational
```

Comment:
Do these have some sort of "official" definition?
For example, what's the gating criteria for moving from `provisional` (which I'm assuming to be WIP) to `implementable`?

What would we use `informational` for? Wouldn't that just be documentation? Or would we record information about a direction we didn't want to go?

Reply:
I'm dropping status as I believe either it is accepted as implementable (merged) or it is rejected (closed). If we need more catagories later...we can add them when it is appropriate.